migration_steps/initialise_environments/delete_existing_data/sql/delete_statements.sql

```diff
@@ -1,29 +1,44 @@
 DROP SCHEMA IF EXISTS deletions CASCADE;
 CREATE SCHEMA deletions;
 
-CREATE TABLE IF NOT EXISTS deletions.base_clients_persons (id int);
-INSERT INTO deletions.base_clients_persons (id)
-SELECT distinct p.id
+-- Create base clients table
+CREATE TABLE IF NOT EXISTS deletions.base_clients_persons (id int, caserecnumber varchar);
+
+-- Insert base clients
+INSERT INTO deletions.base_clients_persons (id, caserecnumber)
+SELECT distinct p.id, p.caserecnumber
 FROM persons p
 WHERE p.type = 'actor_client' and (caseactorgroup <> 'CLIENT-PILOT-ONE' or caseactorgroup is null);
 
 CREATE UNIQUE INDEX stub_person_id_idx ON deletions.base_clients_persons (id);
 
-CREATE TABLE IF NOT EXISTS deletions.pilot_one_feepayers (feepayer_id int);
-INSERT INTO deletions.pilot_one_feepayers (feepayer_id)
+-- Create table for list of deputies to keep
+CREATE TABLE IF NOT EXISTS deletions.pilot_one_deputies (id int);
+
+-- Insert deputies attached to pilot one cases as deputies or feepayers
+INSERT INTO deletions.pilot_one_deputies (id)
+SELECT dep.id
+FROM persons dep
+INNER JOIN order_deputy od ON dep.id = od.deputy_id
+INNER JOIN cases c  ON c.id = od.order_id
+INNER JOIN persons p ON p.id = c.client_id
+WHERE p.type = 'actor_client' and p.caseactorgroup = 'CLIENT-PILOT-ONE'
+UNION
```

Comment:
We need to go and reflect this in the counts work (add in those deps who are feepayers to a CP1 client who may have been missed from the cases relationship)

Reply:
Good point!